ci-operator/step-registry/sandboxed-containers-operator/trustee-install/sandboxed-containers-operator-trustee-install-commands.sh

```diff
@@ -0,0 +1,1387 @@
+TRUSTEE_CN="${TRUSTEE_CN:-kbs-trustee-operator-system}"
+# Set TRUSTEE_CATALOG_SOURCE_NAME to specify the catalog source for operator subscription
+# If not set, try to read from osc-config ConfigMap, otherwise default to "redhat-operators"
+if [ -z "${TRUSTEE_CATALOG_SOURCE_NAME:-}" ]; then
+    # Try to get catalog source name from osc-config ConfigMap created by env-cm step
+    OSC_CONFIG_CATALOG=$(oc get configmap osc-config -n default -o jsonpath='{.data.catalogsourcename}' 2>/dev/null || echo "")
+    if [ -n "$OSC_CONFIG_CATALOG" ]; then
+        TRUSTEE_CATALOG_SOURCE_NAME="$OSC_CONFIG_CATALOG"
+        echo "Using catalog source from osc-config: ${TRUSTEE_CATALOG_SOURCE_NAME}"
+    else
+        TRUSTEE_CATALOG_SOURCE_NAME="redhat-operators"
+        echo "Using default catalog source: ${TRUSTEE_CATALOG_SOURCE_NAME}"
+    fi
+else
+    echo "Using TRUSTEE_CATALOG_SOURCE_NAME environment variable: ${TRUSTEE_CATALOG_SOURCE_NAME}"
+fi
+
+# Function to wait for an operator subscription and CSV to finish installation
+# Parameters:
+#   $1: subscription_name - Name of the subscription
+#   $2: namespace - Namespace where the subscription exists
+#   $3: max_attempts - Maximum number of attempts (default: 60)
+#   $4: sleep_seconds - Seconds to sleep between attempts (default: 10)
+# Returns: 0 if successful, 1 if failed
+# Example:
+#   wait_for_operator_subscription "trustee-operator" "trustee-operator-system"
+#   wait_for_operator_subscription "my-operator" "my-namespace" 120 5
+wait_for_operator_subscription() {
+    local subscription_name="$1"
+    local namespace="$2"
+    local max_attempts="${3:-60}"
+    local sleep_seconds="${4:-10}"
+
+    if [ -z "$subscription_name" ] || [ -z "$namespace" ]; then
+        echo "Error: subscription_name and namespace are required parameters"
+        return 1
+    fi
+
+    # Wait for subscription to reach AtLatestKnown state
+    echo "Waiting for subscription '${subscription_name}' to be ready (state: AtLatestKnown)..."
+    local subscription_ready=false
+
+    for i in $(seq 1 "$max_attempts"); do
+        local subscription_state=$(oc get subscription "${subscription_name}" -n "${namespace}" -o jsonpath='{.status.state}' 2>/dev/null || echo "")
+        if [[ "$subscription_state" == "AtLatestKnown" ]]; then
+            subscription_ready=true
+            echo "Subscription is ready (state: AtLatestKnown)"
+            break
+        fi
+        echo "Waiting for subscription to be ready... (attempt $i/$max_attempts, current state: ${subscription_state:-unknown})"
+        sleep "$sleep_seconds"
+    done
+
+    if [ "$subscription_ready" = false ]; then
+        echo "Warning: Subscription '${subscription_name}' did not reach AtLatestKnown state after $((max_attempts * sleep_seconds)) seconds"
+        echo "Please check the subscription status with: oc get subscription ${subscription_name} -n ${namespace} -o yaml"
```

Comment:
It'd be nice to print the `oc get subscription` as part of the script before exiting.